source/api/ut_runner.pkb

```diff
@@ -137,16 +141,18 @@ create or replace package body ut_runner is
         set(a_test_file_mappings),
         a_client_character_set
       );
+      if a_force_manual_rollback then
+        l_run.set_rollback_type(ut_utils.gc_rollback_manual, a_force=>true);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use either named - or positional - arguments, but not a mix of both. ([plsql:PositionalAndNamedArgumentMixCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3APositionalAndNamedArgumentMixCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=789&issues=AWckVby5d6oxh3R4GXWR&open=AWckVby5d6oxh3R4GXWR)

[ISSUE_KEY]: AWckVby5d6oxh3R4GXWR